setup.cfg

```diff
@@ -42,3 +42,6 @@ ignore =
 
     # Bare except. PyLint finds these for us so we don't need pycodestyle to.
     E722,
+[options.package_data]
```

Comment:
If you want a blank line between the `[pycodestyle]` section in `setup.cfg` (`E722`) and this then `.cookiecutter/includes/setup.cfg` needs to start with a blank line.

(This is true of many of the includes files. I couldn't figure out a better way to implement it. For example I couldn't get it to work exactly right by using Jinja2's "whitespace control" feature. When an include file does exist you'll often want a blank line before its contents but when the same include file does not exist then you don't want to render an extra blank line.)